cli/test/README.md

```diff
@@ -652,32 +654,43 @@ This document includes CLI commands and the expected CLI output in order to help
 |`fastddsqosprof file.xml set transport_descriptor[transport_id].calculate_crc`|**PENDING** (currently `Calculate CRC configuration not yet supported`)|N/A|
 |`fastddsqosprof file.xml set transport_descriptor[transport_id].check_crc`|**PENDING** (currently `Check CRC configuration not yet supported`)|N/A|
 |`fastddsqosprof file.xml set transport_descriptor[transport_id].enable_tcp_nodelay`|**PENDING** (currently `Enable TCP nodelay configuration not yet supported`)|N/A|
 |`fastddsqosprof file.xml set transport_descriptor[transport_id].healthy_check_timeout_ms`|**PENDING** (currently `Healthy check timeout configuration not yet supported`)|N/A|
-|`fastddsqosprof file.xml set transport_descriptor[transport_id].interface_whitelist`|`ERROR: Transport <interface_whitelist> must be keyed []`|`TRANSPORT_USAGE`|
-|`fastddsqosprof file.xml set transport_descriptor[transport_id].interface_whitelist -h`|N/A|`TRANSPORT_USAGE`|
-|`fastddsqosprof file.xml set transport_descriptor[transport_id].interface_whitelist other`|`ERROR: Transport <interface_whitelist> must be keyed []`|`TRANSPORT_USAGE`|
-|`fastddsqosprof file.xml set transport_descriptor[transport_id].interface_whitelist[]`|`ERROR: set command for Transport <interface_whitelist> expects 1 arguments and received 0`|`TRANSPORT_USAGE`|
-|`fastddsqosprof file.xml set transport_descriptor[transport_id].interface_whitelist[].subelement`|`ERROR: Transport <interface_whitelist> must be FINAL element`|`TRANSPORT_USAGE`|
-|`fastddsqosprof file.xml set transport_descriptor[transport_id].interface_whitelist[] -h`|N/A|`TRANSPORT_USAGE`|
-|`fastddsqosprof file.xml set transport_descriptor[transport_id].interface_whitelist[] --help`|N/A|`TRANSPORT_USAGE`|
-|`fastddsqosprof file.xml set transport_descriptor[transport_id].interface_whitelist[] help`|N/A|`TRANSPORT_USAGE`|
-|`fastddsqosprof file.xml set transport_descriptor[transport_id].interface_whitelist[] arg1 arg2`|`ERROR: set command for Transport <interface_whitelist> expects 1 arguments and received 2`|`TRANSPORT_USAGE`|
-|`fastddsqosprof file.xml set transport_descriptor[transport_id].interface_whitelist[] arg1 arg2 -h`|N/A|`TRANSPORT_USAGE`|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].interface_whitelist`|`ERROR: Transport <interface_whitelist> must be keyed []`|`TRANSPORT_WHITELIST_INTERFACE_USAGE`|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].interface_whitelist -h`|N/A|`TRANSPORT_WHITELIST_INTERFACE_USAGE`|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].interface_whitelist other`|`ERROR: Transport <interface_whitelist> must be keyed []`|`TRANSPORT_WHITELIST_INTERFACE_USAGE`|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].interface_whitelist[]`|`ERROR: set command for Transport <interface_whitelist> expects 1 arguments and received 0`|`TRANSPORT_WHITELIST_INTERFACE_USAGE`|
```

Comment:
This case is showing the `TRANSPORT_USAGE` instead of `TRANSPORT_WHITELIST_INTERFACE_USAGE`. The test plan is correct but the current behavior is not correct.